docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Algolia Personalization
+description: Find out how you can enable Algolia personalization in your Spryker shop
+last_updated: Nov 24, 2024
+template: howto-guide-template
+---
+
+{% info_block infoBox "Info" %}
+
+Default Spryker installation supports Algolia personalization only for YVES frontend.
+If you plan to use Algolia personalization in headless frontend or mobile application follow [this guide](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/algolia-personalization-headless.html). 
+
+This feature also enables other Algolia premium features:
+
+- Dynamic Re-Ranking
+- Query Categorization
+- Search analytics
+- Revenue analytics
+- A/B Testing
+
+{% endinfo_block %}
+
+## Prerequisites
+
+Your shop already has [integrated](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/integrate-algolia.html) 
+and [configured](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/configure-algolia.html) ACP Algolia App
+and your Algolia search indexes have products. 
+
+## Update Spryker Shop
+
+
+### Update Spryker packages
+
+```bash
+composer update --with-dependencies spryker/kernel-app spryker/search-http spryker/customer \
+spryker-shop/cart-page spryker-shop/catalog-page spryker-shop/checkout-page spryker-shop/home-page spryker-shop/payment-page \
+spryker-shop/product-detail-page spryker-shop/product-group-widget spryker-shop/product-review-widget \
+spryker-shop/product-set-detail-page spryker-shop/quick-order-page spryker-shop/shop-ui
+```
+
+### Install new Spryker packages
+
+```bash
+composer install spryker-shop/traceable-event-widget
+```
+
+
+### Minimum package versions
+
+```bash
+spryker/kernel-app: ^1.2.0
+spryker/search-http: ^0.5.0
+spryker/customer: ^7.61.0
+spryker-shop/cart-page: ^3.45.0
+spryker-shop/catalog-page: ^1.28.0
+spryker-shop/checkout-page: ^3.32.1
+spryker-shop/home-page: ^1.2.0
+spryker-shop/payment-page: ^1.5.0
+spryker-shop/product-detail-page: ^3.22.1
+spryker-shop/product-group-widget: ^1.10.1
+spryker-shop/product-review-widget: ^1.16.1
+spryker-shop/product-set-detail-page: ^1.11.0
+spryker-shop/quick-order-page: ^4.10.0
+spryker-shop/shop-ui: ^1.79.0
+spryker-shop/traceable-event-widget: ^1.0.1
```

Comment:
@vol4onok it can be merged with the command above `composer update --with-dependencies spryker/kernel-app:'^1.2.0' spryker/search-http:^0.5.0 ...`

Reply:
I moved it